USAGE.md

```diff
@@ -0,0 +1,105 @@
+# Amazon Alexa Smart Home Skill for openHAB 2
+
+Amazon certified openHAB2 as a [smarthome skill for Alexa](https://www.amazon.com/openHAB-Foundation/dp/B01MTY7Z5L).
+This page describes how to use the skill certified by Amazon.
+This skill allows you to connect your openHAB setup through the myopenHAB.org cloud service to Amazon Alexa.
+(See [README.md](https://github.com/openhab/openhab-alexa/blob/master/README.md) for other setup-options regarding your own openHAB-cloud.)
```

Comment:
"setup options", the "Setup-Optionen" dash is something we use in german only ;)

Reply:
ok